content/tutorials/reactor-event-cloudflare.textile

```diff
@@ -0,0 +1,256 @@
+---
+title: "Reactor Events Tutorial: Building A Realtime 2D Game with React"
+alt_title: "Building A Realtime 2D Game with React"
+excerpt: "Learn how to use our Reactor Functions to consume realtime data with Cloudflare"
+section: tutorials
+category: external-services
+index: 100
+platform: browser
+authors:
+- author_name: Abraham Jr. Agiri
+  author_bio: "I'm a software developer who loves writing and in love with open source"
+  author_profile_url: https://github.com/codeekage
+  author_image: ://avatars3.githubusercontent.com/u/24491408?s=400&v=4
+level: medium
+reading_time: 10
+languages:
+  - javascript
+  - nodejs
+ably_product: reactor
+tags:
+  - ably-features
+---
+
+The Ably Reactor provides a way to trigger events and to stream data from Ably's pub/sub channels. This turorial will go over how to use the Ably Reactor with "Cloudflare workers":https://developers.cloudflare.com/workers/quickstart/. 
```

Comment:
Same comments as other PR in regards to this - see https://github.com/ably/docs/pull/820/files